src/platform/silabs/wifi/SiWx/siwx917/netinet/in.h

```diff
@@ -0,0 +1 @@
+#include <netinet_in.h>
```

Comment:
It would be useful to add comments explaining why this file structure is needed and that is why we endup haveing a header with only 1 include